src/viam/sdk/CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ target_sources(viamsdk
     common/linear_algebra.cpp
     common/pose.cpp
     common/proto_type.cpp
+    common/service_helper.cpp
```

Comment:
Currently, I'm not publishing the `service_helper.hpp` header, so `ServiceHelper` is effectively a private utility of the SDK and only we can use it to implement our services. If we want others (module authors) to also have access to it, we should publish the header.

Reply:
I would be in favor of exposing it for module authors. When I was writing the complex module example I definitely looked at the existing server implementations for inspiration; it would be frustrating if those "examples" were leveraging utilities I didn't have access to.

Reply:
> We could keep it private for now until we have more experience with it, and then release it.

SGTM especially if we're introducing this for only a single component and service for the time being.